index.html

```diff
@@ -189,5 +184,58 @@ <h5>Skills</h5>
         </li>
       </ul>
     </section>
+
+    <!--forms section-->
+
+    <div class="form-container">
+      <p class="form-intro">
```

Comment:
- [ ] I suggest using semantic HTML5 tag such as `<section class="form-container">` instead of  `<div class="form-container">` to follow the [best practices for HTML](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 💡